lib/types/front-chat-types/index.ts

```diff
@@ -12,11 +12,32 @@ declare global {
  * Types.
  */
 
+export enum FrontChatCmdType {
+  Init = 'init',
+  Show = 'show',
+  Shutdown = 'shutdown'
+}
+
+export type FrontChatParams = {
+  chatId?: string;
+  userId?: string;
+  userHash?: string;
+  shouldShowWindowOnLaunch?: boolean;
+  shouldExpandOnShowWindow?: boolean;
+  shouldHideCloseButton?: boolean;
+  shouldHideExpandButton?: boolean;
+  welcomeMessageAppearance?: 'hidden' | 'always';
+  isMobileWebview?: boolean;
+  useDefaultLauncher?: boolean;
+  onInitCompleted?: () => void;
+  // Allow any other key with a value of string, boolean, object, or undefined
+  [key: string]: string | boolean | object | undefined;
```

Comment:
Can it be `number` as well? If so, maybe `unknown` is a better fit.

```suggestion
  [key: string]: unknown;
```